model/trace/database.yaml

```diff
@@ -358,7 +358,7 @@ groups:
         examples: ['cosmos-netstandard-sdk/3.23.0\|3.23.1\|1\|X64\|Linux 5.4.0-1098-azure 104 18\|.NET Core 3.1.32\|S\|']
       - ref: db.cosmosdb.connection_mode
         requirement_level:
-          conditionally_required: if not `direct` (or pick gw as default)
+          conditionally_required: if not `direct` (the default value is assumed to be `direct`).
```

Comment:
Direct is only default for .NET and Java, and direct mode doesn't exist in other languages. Should we say conditionally required if using .NET or Java?

Reply:
good point. If it's only required for .NET and Java, then the default should be `gateway`. 

The only reason to have a default is to slightly reduce telemetry volume. So if we pick gateway as default, most of .NET and Java users will have `direct` populated.
If `direct` is default, then all python, js, go, etc will have the attribute.

wdyt?